src/components/result-list/student-result.tsx

```diff
@@ -24,37 +24,17 @@ export const StudentResults = ({ student, language, studentBaseUrl }: StudentRes
         return gradePoints
       }
 
-      return (
-        student.includedExams
-          // if exam uuid is in overlappingExamUuids, we have already counted it towards the total
-          .filter(exam => !overlappingExamUuids.includes(exam.examUuid))
-          .reduce((total, exam) => exam.details[0].gradePoints + total, gradePoints)
-      )
+      return student.includedExams
+        .filter(exam => {
+          const examUuidsWithBestGrades = examsWithBestGrades.map(exam => exam.examUuid)
```

Comment:
Magiaa! Käytetään examsWithBestGrades muuttujaa ennen sen alustusta. Ainakin siltä vaikuttaa kun lukee ylhäältä alaspäin. Pitäisikö se muuttuja passata tähän funkkariin?

Reply:
Joo, nyt kun lukee niin onhan toi aika epäselvää. Lisäsin korjauksen.